spec/features/test_for_ci_spec.rb

```diff
@@ -1,4 +1,9 @@
 require "rails_helper"
+require "axe-rspec"
+
+RSpec.configure do |c|
+  c.filter_run_excluding broken: true
```

Comment:
I am not sure `broken` is the right name here? These are going to be accessibility test so perhaps `accessibility: true` would be better?